frontend/device/android/device.lua

```diff
@@ -86,6 +111,7 @@ function Device:init()
                 or ev.code == C.APP_CMD_WINDOW_REDRAW_NEEDED then
                 this.device.screen:_updateWindow()
             elseif ev.code == C.APP_CMD_RESUME then
+                this.device.getExternalDictLookupList = getExternalDicts
```

Comment:
What does that do?
It looks like you just re-assigning that function to the that name, and I see no reason for getExternalDictLookupList to have been assigned something else in between (?).

Reply:
it updstes the list of availab

Reply:
But you're just re-assigning that function to that name, not actually calling it. So that's a no-op.
And you're actually calling it in `doExternalDictLookup `, so doing the `android.isPackageEnabled` at each lookup (as you got rid of my `EXTERNAL_DICTS_AVAILABILITY_CHECKED` flag).
If you really care about rechecking availability, you should keep that kind of flag (to avoid rechecking on each lookup in the same foreground session), and just unset it onResume, so the next lookup (only) do the availability check.
(Or do I miss something?)